docs/command_line_tool/alternate_install.md

```diff
@@ -1,9 +1,11 @@
 ---
-title: Building
+title: Alternate Installation Methods
 category: 6255e67693d5200013b1fa41
```

Comment:
```suggestion
category: 6255e67693d5200013b1fa3e
```
Given that the location of this file changed from the `docs/knowledge_base` directory to the `docs/command_line_tool` directory, the assumption is that this page should fall under the `COMMAND LINE TOOL` category.